WordPress/src/main/java/org/wordpress/android/widgets/WPScrollView.java

```diff
@@ -35,37 +44,8 @@ public void setScrollDirectionListener(ScrollDirectionListener listener) {
     @Override
     public boolean onTouchEvent(MotionEvent event) {
         if (mScrollDirectionListener != null) {
```

Comment:
Just want to double check that the following `null` check is correct:
```
        if (mScrollDirectionListener != null) {
            mDetector.onTouchEvent(event);
        }
```

I think yes, since `mDetector` is always not null, and it fires updates on `mScrollDirectionListener`.